docs/tables/aws_elasticache_parameter_group.md

```diff
@@ -0,0 +1,33 @@
+# Table: aws_elasticache_parameter_group
+
+Parameter groups are collection of parameters which control the behavior of the ElastiCache clusters.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  cache_parameter_group_name,
+  description,
+  cache_parameter_group_family,
+  description,
+  is_global
+from
+  aws_elasticache_parameter_group;
+```
+
+
+### List parameter groups in the undesired (for example, redis5.0 and memcached1.5 are desired) family types.
```

Comment:
List parameter groups in the undesired (for example, redis5.0 and memcached1.5 are desired) family types. >>  This can be in general as below

List parameter groups, which are not family type of redis5.0 and memcached1.5.